### ISSUE TITLE
Increase coverage of metric alias of `LightGBMTuner`

### ISSUE THREAD

**Issue by toshihikoyanase** [optuna/optuna#1806]
In #1602, the alias of `l2` has been supported by `LightGBMTuner`. However, the original `lightgbm` has more metric aliases as can be seen in [ParseMetricAlias](https://github.com/microsoft/LightGBM/blob/083b02af73b8fcf9a92af78612c6cca6243df752/include/LightGBM/config.h#L1111-L1112), and some of them are not supported by `LightGBMTuner`.

Please add the aliases of the following metrics to [`_ALIAS_METRIC_LIST`](https://github.com/optuna/optuna/blob/master/optuna/integration/_lightgbm_tuner/alias.py#L43):

- [x] `l2`
- [x] `rmse`
- [x] `l1`
- [x] `binary_logloss`
- [x] `ndcg`
- [x] `map`
- [x] `multi_logloss`
- [x] `cross_entropy`
- [x] `cross_entropy_lambda`
- [x] `kullback_leibler`
- [x] `mape`
- [x] `auc_mu`
- [x] `custom`

Please note that `custom` needs an additional fix as mentioned in https://github.com/optuna/optuna/issues/1351.

**Comment by toshihikoyanase** [comment]
Let me close this issue since #1807 and #1822 resolved it.
Thank you for your contribution, @upura and @nyanhi !
